### PR TITLE
SONARUNNER-146 Expose enhanced issues location in issues mode

### PR DESCRIPTION
**Pull request by dbmeneses**

### CODE REVIEW

File: sonar-runner-batch/src/main/java/org/sonar/runner/batch/BatchIsolatedLauncher.java [hunk 1]
```diff
@@ -35,15 +37,18 @@
  * the same version of sonar-batch as the server.
  */
 public class BatchIsolatedLauncher implements IsolatedLauncher {
+  private final static String VERSION_FORMAT = "^(\\d+)\\.(\\d+)";
```
Comment (SonarTech) [inline sonar-runner-batch/src/main/java/org/sonar/runner/batch/BatchIsolatedLauncher.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by dbmeneses** [comment]
LGTM

File: sonar-runner-batch/src/test/java/org/sonar/runner/batch/BatchIsolatedLauncherTest.java [hunk 2]
```diff
@@ -51,6 +52,23 @@ public void setUp() {
   }
 
   @Test
+  public void supportsPreciseLocation() {
```
Comment (henryju) [inline sonar-runner-batch/src/test/java/org/sonar/runner/batch/BatchIsolatedLauncherTest.java]: Does it works with versions like 5.3-SNAPSHOT or 5.3-RC1?

**Comment by henryju** [comment]
LGTM except version regexp (see my comment)

**Pull request merged**